### ISSUE TITLE
toFormat() displays zero-precision currencies with decimals, like 15.00 instead of 15

### ISSUE THREAD

**Issue by risafj** [dinerojs/dinero.js#203]
**Describe the bug**
Hi 👋🏻   First of all, thank you for creating/maintaining this cool library!
I'm not sure if this is a bug, but I'm reporting it as one because I think it's unintended behavior.

I'm trying to use dinero.js for a multi-currency app I'm developing. One of the currencies I handle is JPY, which has no sub-unit. A JPY amount should look like `¥500` (this is how `toLocaleString` formats it).

But when I use dinero.js, it's formatted like`¥500.00`, which looks very unnatural because JPY is never represented this way.

```ts
// ¥500.00
Dinero({ amount: 500, currency: 'JPY', precision: 0 }).toFormat()
```

I know I can pass a format to `toFormat()` like below:

```ts
// ¥500
Dinero({ amount: 500, currency: 'JPY', precision: 0 }).toFormat('$0,0')
```

But then, this format cannot be used for other currencies that have sub-units like USD and EUR. My current workaround is to manually handle formats depending on the currency, like this:

```ts
const amountToDisplayFormat= (amount: number, currency: string) => {
   // These would have to be modified if I add more currencies that have precision other than 2
  const precision = currency === 'JPY' ? 0 : 2
  const format = currency === 'JPY' ? '$0,0' : '$0,0.00'

  return Dinero({
    amount,
    currency,
    precision
  }).toFormat(format)
}
```

TLDR: Currencies that don't have 2 decimal places don't seem to be formatted correctly.

**To reproduce**

The code snippets above basically explain how to reproduce this.

**Expected behavior**
The code below should return `¥500`, not  `¥500.00`.
```ts
Dinero({ amount: 500, currency: 'JPY', precision: 0 }).toFormat()
```

**Screenshots**

**Browser**

- OS: macOS
- Browser: N/A
- Version: N/A

**Comment by Akhouad** [comment]
I see no harm using the workaround you suggested tbh.
